android/titanium/src/java/org/appcelerator/titanium/proxy/RProxy.java

```diff
@@ -21,8 +21,11 @@
 {
 	private static final String TAG = "TiAndroidRProxy";
 
-	private static final String[] RESOURCE_TYPES = { "anim", "array",   "attr",   "color",  "dimen", "drawable",
-													 "id",   "integer", "layout", "string", "style", "styleable" };
+	private static final String[] RESOURCE_TYPES = { "anim",    "animator",     "array",    "attr",     "bool",
+													 "color",   "dimen",        "drawable", "fraction", "id",
+													 "integer", "interpolator", "layout",   "menu",     "mimap",
+													 "plurals", "raw",          "string",   "style",    "styleable",
+													 "xml" };
```

Comment:
I think "mimap" is a typo. Should be "mipmap", right?

"mipmap" is intended to be used to store app/launcher icons and is intended to replace the "drawable" folder for this purpose.

Reply:
Good catch! Thats a typo, will resolve after vacation.